integration-tests/gradle/src/test/resources/jandex-basic-multi-module-project/gradle.properties

```diff
@@ -0,0 +1,4 @@
+quarkusPlatformArtifactId=quarkus-bom
+quarkusPlatformGroupId=io.quarkus
+quarkusPlatformVersion=999-SNAPSHOT
+quarkusPluginVersion=999-SNAPSHOT
```

Comment:
The last two lines should be removed from this file. These versions will be injected as part of the test setup. We can't be mentioning any version in here otherwise the release process (changing the versions) will fail.

Reply:
Okk